include/openssl/md2.h

```diff
@@ -19,32 +19,37 @@
 # include <openssl/opensslconf.h>
 
 # ifndef OPENSSL_NO_MD2
-# include <stddef.h>
-# ifdef  __cplusplus
+#  if !defined(OPENSSL_NO_DEPRECATED_3_0)
+#   include <stddef.h>
+#   ifdef  __cplusplus
 extern "C" {
-# endif
+#   endif
```

Comment:
I would put the deprecation guard after this. That will make things slightly less ugly

Reply:
This slightly changes the semantics by including `stddef.h`.  Should be harmless I know.

Reply:
Which is what this PR did and shouldn't have.  I'll fix.